assets/src/scripts/selectors/timeSeriesSelector.spec.js

```diff
@@ -152,76 +152,130 @@ describe('timeSeriesSelector', () => {
     describe('hasTimeSeries', () => {
         const TEST_STATE = {
             series: {
+                variables: TEST_VARS,
                 requests : {
-                    'current': {},
+                    'current:P7D': {},
                     'median' : {},
                     'current:P30D:00060': {}
                 }
+            },
+            timeseriesState: {
```

Comment:
Should this be changed to `timeSeriesState` in the tests and in the store? Not sure if we want this consistency to extend into the store as well. :thinking: